package.json

```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://ichbineinekartoffel.de",
   "devDependencies": {
+    "cssnano": "^4.1.10",
```

Comment:
This guy as well, what is the reason behind it?

Reply:
When installed Parcel with was added to the package.